doc/input-output-reference/src/overview/group-variable-refrigerant-flow-equipment.tex

```diff
@@ -782,64 +782,63 @@ \subsubsection{Inputs}\label{inputs-1-048}
 
 \paragraph{Field: Minimum Outdoor Air Temperature in Cooling Mode}\label{field-minimum-outdoor-air-temperature-in-cooling-mode}
 
-This numeric field defines the minimum outdoor temperature allowed for cooling operations. Below this temperature, cooling is disabled. If this field is left blank, the default value is -6ºC.
- If this field is left blank, the default value is -20ºC.
+This numeric field defines the minimum outdoor temperature allowed for cooling operations. Below this temperature, cooling is disabled. If this field is left blank, the default value is -6$^\circ$C.
 
 \paragraph{Field: Maximum Outdoor Air Temperature in Cooling Mode}\label{field-maximum-outdoor-air-temperature-in-cooling-mode}
 
-This numeric field defines the maximum outdoor temperature allowed for cooling operations. Above this temperature, heating is disabled. If this field is left blank, the default value is 16ºC.
+This numeric field defines the maximum outdoor temperature allowed for cooling operations. Above this temperature, heating is disabled. If this field is left blank, the default value is 43$^\circ$C.
 
 \paragraph{Field: Minimum Outdoor Air Temperature in Heating Mode}\label{field-minimum-outdoor-air-temperature-in-heating-mode}
 
-This numeric field defines the minimum outdoor temperature allowed for heating operation. Below this temperature, heating is disabled. If this field is left blank, the default value is -20ºC.
+This numeric field defines the minimum outdoor temperature allowed for heating operation. Below this temperature, heating is disabled. If this field is left blank, the default value is -20$^\circ$C.
 
 \paragraph{Field: Maximum Outdoor Air Temperature in Heating Mode}\label{field-maximum-outdoor-air-temperature-in-heating-mode}
 
-This numeric field defines the maximum outdoor temperature allowed for heating operation. Above this temperature, heating is disabled. If this field is left blank, the default value is 16ºC.
+This numeric field defines the maximum outdoor temperature allowed for heating operation. Above this temperature, heating is disabled. If this field is left blank, the default value is 16$^\circ$C.
 
 \paragraph{Field: Outdoor Unit Reference Superheating}\label{field-outdoor-unit-reference-superheating}
 
-This numeric field defines the reference superheating degrees of the outdoor unit. If this field is blank, the default value of 3.0ºC is used.
+This numeric field defines the reference superheating degrees of the outdoor unit. If this field is blank, the default value of 3.0$^\circ$C is used.
 
 \paragraph{Field: Outdoor Unit Reference Subcooling}\label{field-outdoor-unit-reference-subcooling}
 
-This numeric field defines the reference subcooling degrees of the outdoor unit. If this field is blank, the default value of 3.0ºC is used.
+This numeric field defines the reference subcooling degrees of the outdoor unit. If this field is blank, the default value of 5.0$^\circ$C is used.
 
```

Comment:
Check with IDD and got the correct value.

Reply:
That's correct. But the name of these fields is actually "Reference Outdoor Unit *"

![image](https://user-images.githubusercontent.com/5778156/190002486-a3f83506-6d29-470d-a8d4-e57f27bea205.png)

Reply:
I will stop making recommendations since these changes are many and too hard to provide a thorough review.

Reply:
Thanks! Changed these ones for the VRF AC to reflect the IDD specifications.